### DISCUSSION TITLE
Reviewing Reviews

### DISCUSSION THREAD

**Discussion by micahvandegrift** [knowledgefutures/pubpub#1811] [Support]
Hi Pubbers - we're in the middle of releasing [Fermentology 2.0](fermentology.pubpub.org), and as the self-appointed EiC/Final filter I just wanted to call out a frustration on Reviews...

Why, when a review is Closed or Completed, does it remain on the Reviews page/queue? And, what is the organizing principle for how they display on the page? New releases don't seem to group at the top of the page or really in any order that I can understand.

Basically, I'm tired of having the scroll 2-3 times to find that one green review to release. I'd love a sort function like the Pubs vs. Collections clicker on the Community Overview page, so that I could click off Completed and Closed and just see the Open reviews to plow through in one sit down. Or better yet, a way to Close and Archive a review so that I never have to see it again but it is logged in there somewhere. 

Hope this is helpful!

**Comment by writingpoli** [comment]
My understanding is that Reviews is undergoing a complete redo. Hit up help@pubpub.org to be part of discussions. 

Otherwise, I completely agree with you. Reviews is largely useless currently.
